sdk/tables/azure-tables/src/serialization.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { base64Encode, base64Decode } from "./utils/bufferSerializer";
+
+const Edm = {
+  Binary: "Edm.Binary",
+  Boolean: "Edm.Boolean",
+  DateTime: "Edm.DateTime",
+  Double: "Edm.Double",
+  Guid: "Edm.Guid",
+  Int32: "Edm.Int32",
+  Int64: "Edm.Int64",
+  String: "Edm.String"
+};
```

Comment:
```suggestion
} as const;
```
would give you slightly stronger typing here